beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/DefaultPerformanceTracker.java

```diff
@@ -175,23 +164,13 @@ public void onSlot(UInt64 slot) {
     }
   }
 
-  private BlockPerformance getBlockPerformanceForEpochs(
-      UInt64 startEpochInclusive, UInt64 endEpochExclusive) {
+  private BlockPerformance getBlockPerformanceForEpoch(UInt64 currentEpoch) {
     int numberOfBlockProductionAttempts =
-        blockProductionAttemptsByEpoch
-            .subMap(startEpochInclusive, true, endEpochExclusive, false)
-            .values()
-            .stream()
-            .mapToInt(AtomicInteger::get)
-            .sum();
+        blockProductionAttemptsByEpoch.values().stream().mapToInt(AtomicInteger::get).sum();
```

Comment:
I think we should be getting the value for the epoch we're interested in, not just summing all epochs.  Most of the time there will only be one epoch but depending on timing of requests that's not always entirely guaranteed.